src/main/java/org/motechproject/mots/dto/JasperTemplateDto.java

```diff
@@ -40,5 +41,6 @@
 
   @Getter
   @Setter
+  @Valid
```

Comment:
if JasperTemplateParameterDto has no validation, there is no sense of adding valid here